bot/constants.py

```diff
@@ -8,3 +8,6 @@
 GPT_MODEL = 'gpt-3.5-turbo'
 ORGANIZATION_ID = 'org-XwMNyaooe9J64WbgekZen3L0'
 PROJECT_ID = 'proj_GZHV06BPXtu8fHt2xwoEuhmp'
+
+# Telegram Token
+BOT_TOKEN = '7144215394:AAGJXRA-PYsuczsq-AiPmLuxVBKkdM27lIw'
```

Comment:
BOT_TOKEN should not be displayed and pushed to the remote repository due to security reasons. Assign the value of this variable to 'BOT_TOKEN' (as I've done it with OPENAI_API_KEY) and save the token itself somewhere locally for yourself, to past it later whenever you need to use it (but without pushing these changes to remote repository)